README.md

```diff
@@ -30,6 +30,10 @@ This is a top-down LL parser that parses directly into Zig structs.
   * [x] Mapping to optional fields
   * [x] Mapping to HashMaps
 * [ ] Serialization
+    * [x] Basic types like `int`s, floating points, strings, booleans etc.
+    * [x] Arrays
+    * [x] Tables
```

Comment:
Let's name it `top level tables`, because IIUC nested tables and array of tables are not implemented yet.